public/app/features/admin/UserListPublicDashboardPage/DashboardsListModalButton.tsx

```diff
@@ -18,15 +19,24 @@ export const DashboardsListModal = ({ email, onDismiss }: { email: string; onDis
   const { data: dashboards, isLoading } = useGetActiveUserDashboardsQuery(email);
 
   return (
-    <Modal className={styles.modal} isOpen title="Public dashboards" onDismiss={onDismiss}>
+    <Modal
+      className={styles.modal}
+      isOpen
+      title={t('dashboards-list-modal-button.public-dashboard.modal-title', 'Public dashboards')}
+      onDismiss={onDismiss}
+    >
       {isLoading ? (
         <div className={styles.loading}>
-          <LoadingPlaceholder text="Loading..." />
+          <LoadingPlaceholder text={t('dashboards-list-modal-button.public-dashboard.loading-text', 'Loading...')} />
         </div>
       ) : (
         dashboards?.map((dash) => (
           <div key={dash.dashboardUid} className={styles.listItem} data-testid={selectors.listItem(dash.dashboardUid)}>
-            <p className={styles.dashboardTitle}>{dash.dashboardTitle}</p>
+            <p className={styles.dashboardTitle}>
+              <Trans i18nKey="dashboards-list-modal-button.public-dashboard.dashboard-title">
+                {{ dashboardTitle: dash.dashboardTitle }}
+              </Trans>
```

Comment:
This doesn't do anything - user input isn't translated. This should remain as it was before.